### ISSUE TITLE
Support SVGs as background images in Social cards

### ISSUE THREAD

**Issue by willmcgugan** [squidfunk/mkdocs-material#5486]
### Description

It would be great if the social cards supported SVGs as background images. This will be particularly helpful for our blog, which typically have SVG images.

### Related links

https://twitter.com/squidfunk/status/1655627179873513500

### Use Cases

Blog posts with SVG images

### Before submitting

- [X] I have read and followed the [change request guidelines](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/).
- [X] I have verified that [my idea is a change request and not a bug report](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/#its-not-a-bug-its-a-feature).
- [X] I have ensured that, to the best of my knowledge, [my idea will benefit the entire community](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/#benefit-for-the-community).
- [X] I have included relevant links to [the documentation](https://squidfunk.github.io/mkdocs-material), related [issues](https://github.com/squidfunk/mkdocs-material/issues), and [discussions](https://github.com/squidfunk/mkdocs-material/discussions) to underline the need for my idea.

**Comment by squidfunk** [comment]
Thanks for suggesting! Indeed a very reasonable request. [`e3c253f`](https://github.com/squidfunk/mkdocs-material-insiders/commit/e3c253fd455dd7e7afcf9f2b3dcfec24ee816496) adds support for SVG backgrounds.

Example:

``` yaml
size: { width: 1200, height: 630 }
layers:
  - background:
      image: image.jpg
  - background:
      image: image.svg
```

Just pick any SVG image, it will be scaled to cover.

**Comment by squidfunk** [comment]
Released as part of [9.1.11+insiders-4.33.1](https://squidfunk.github.io/mkdocs-material/insiders/changelog/#4.33.1)
